libs/interpret-vision/src/lib/VisionExplanationDashboard/VisionExplanationDashboard.tsx

```diff
@@ -220,37 +221,42 @@ export class VisionExplanationDashboard extends React.Component<
   ): void => {
     this.setState({ searchValue: newValue || "" });
   };
-  private onItemSelect = (item: IVisionListItem): void => {
-    this.setState({ panelOpen: !this.state.panelOpen, selectedItem: item });
-    const index = item.index;
+  private onItemSelect = (item: IVisionListItem, selectedObject = -1): void => {
```

Comment:
in which case will selectedObject be -1?

Reply:
In another word, will all items have label with prefix "Object:"?

Reply:
All items should have prefix "Object: "!

I had the default value be "-1" because if an option from the drop down has not been selected yet, then the explanation should not be retrieved. In other words, the -1 indicates that the object within the drop down has not been selected it. 

LMK if that makes sense/if I can add more documentation to make that more clear! Or if there is a more clear way of doing this :)